variables.tf

```diff
@@ -19,6 +19,12 @@ variable "region" {
   description = "The IBM Cloud region where the cluster will be provisioned."
 }
 
+variable "private_environment" {
+  type        = bool
+  description = "Enable if using private schematics agents."
```

Comment:
It should probably not mention schematics agents here. Thats not a requirement. Its just the runtime has to have access to private endpoint